modules/apps/foundation/configuration-admin/configuration-admin-action/bnd.bnd

```diff
@@ -0,0 +1,6 @@
+Bundle-Name: Liferay Configuration Admin Action
```

Comment:
Let's call the module configuration-admin-api so that it can be used to host more extension points and APIs later on.